.github/auto_assign.yml

```diff
@@ -0,0 +1,49 @@
+# Set to true to add reviewers to pull requests
+addReviewers: true
+
+# Set to true to add assignees to pull requests
+addAssignees: false
+
+# A number of reviewers added to the pull request
+# Set 0 to add all the reviewers (default: 0)
+numberOfReviewers: 2
+
+# A number of assignees to add to the pull request
+# Set to 0 to add all of the assignees.
+# Uses numberOfReviewers if unset.
+# numberOfAssignees: 2
+
+# Set to true to add reviewers from different groups to pull requests
+useReviewGroups: true
+
+# A list of reviewers, split into different groups, to be added to pull requests (GitHub user name)
+reviewGroups:
+  maintainers:
```

Comment:
super nit: is there some way to generate this file, such that there's a single source of truth for these values?

Reply:
I don't think the GitHub action supports groups/owners files unfortunately.

Reply:
agree this would be nice but 🤷.  Adding a note to also check this file when updating the main list might make easier to remember